### ISSUE TITLE
New russian language model

### ISSUE THREAD

**Issue by Desklop** [bakwc/JamSpell#69]
Здравствуйте, я тут обучил JamSpell на наборе данных из русских субтитров (примерно 5 млн предложений) к 347 различным сериалам, взятыми из корпуса [Taiga](https://tatianashavrina.github.io/taiga_site/downloads). После очистки данных получился текстовый файл с субтитрами размером 310Mb. 

Результаты тестирования модели:
![result_evaluate](https://user-images.githubusercontent.com/37866973/49441090-84549d80-f7d6-11e8-9f5e-a736f9c6b3e8.png)

Скачать можно из моего Google Drive: [ru_model_subtitles.zip](https://drive.google.com/open?id=1_IIVslK126ehUEYTC1DtIBPlHFaq6o0x) (58Mb)

PS. Буду только рад, если добавите эту модель к списку готовых моделей в README проекта :)

**Comment by olegtarasov** [comment]
@Desklop Владислав, здравствуйте! Большое спасибо за модель! Не могли бы выложить заодно и корпус для обучения?

**Comment by Desklop** [comment]
@olegtarasov да, в моём репозитории вы можете найти всю необходимую информацию: [Russian subtitles dataset](https://github.com/Desklop/Russian_subtitles_dataset)

**Comment by olegtarasov** [comment]
@Desklop Круто, спасибо! Поисследуем :)

**Comment by bakwc** [comment]
Посмотрите так же на `Pro` версию, она включает в себя большую модель (~2 Гб. данных) обученную на википедии, новостях, субтитрах (не только из taiga, но и из opensubtitles.org), литературе. Исходные данные так же доступны.
